x-pack/plugins/apm/public/utils/formatters.js

```diff
@@ -52,3 +52,14 @@ export function asInteger(value) {
 export function tpmUnit(type) {
   return type === 'request' ? 'rpm' : 'tpm';
 }
+
+export function getDurationPercent(
+  itemDuration: number,
+  totalDuration?: number,
+  fallbackResult?: string
```

Comment:
Fallback could probably be a number too ... maybe just `any` here?

Reply:
Should we restrict it to `string | number`? If someone passes in an object or array it's probably a mistake we want to catch.

Reply:
... changed my mind on this. See my suggestion above.